src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShippingInformation.tsx

```diff
@@ -1,28 +1,30 @@
+import { ActionType, type ClickedOnLearnMore } from "@artsy/cohesion"
 import { Spacer, Text } from "@artsy/palette"
+import { ArtsyShippingEstimate } from "Components/ArtsyShippingEstimate"
+import { RouterLink } from "System/Components/RouterLink"
 import type { ArtworkSidebarShippingInformation_artwork$data } from "__generated__/ArtworkSidebarShippingInformation_artwork.graphql"
 import { createFragmentContainer, graphql } from "react-relay"
-
-import { ActionType, type ClickedOnLearnMore } from "@artsy/cohesion"
-import { RouterLink } from "System/Components/RouterLink"
 import { useTracking } from "react-tracking"
 
 export interface ShippingInformationProps {
   artwork: ArtworkSidebarShippingInformation_artwork$data
 }
+const CALCULATED_IN_CHECKOUT_MESSAGE = "Shipping: Calculated in checkout"
```

Comment:
This is not the case any more after this pr: https://github.com/artsy/metaphysics/pull/6383

Reply:
Removing for now bc designs don't need it